diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -412,7 +468,178 @@ def main():
+        except FileNotFoundError as e:
+            save_fail_message = "\nUnable to create directory"
+            print(save_fail_message)
+            parser.custom_error(str(e))
+
+    # Morph morph_file against all other files in target_directory
+    results = {}
+    for target_file in target_list:
+        if target_file.is_file:
+            # Set the save file destination to be a file within the SLOC directory
+            if save_opt is not None:
+                opts.saveloc = Path(save_morphs_here).joinpath(save_names.get(target_file.name).get("save_morph_as"))
+            # Perform a morph of morph_file against target_file
+            pargs = [morph_file, target_file]
+            results.update({
+                target_file.name:
+                    single_morph(parser, opts, pargs, stdout_flag=False),
+            })
+
+    # Parse rws
+    file_names = []
+    rws = []
+    pearsons = []
+    scales = []
+    stretches = []
+    smears = []
+    results_length = len(results.keys())
+    for key in results.keys():
+        file_names.append(key)
+        rws.append(results[key]["Rw"])
+        pearsons.append(results[key]["Pearson"])
+        if "scale" in results[key]:
+            scales.append(results[key]["scale"])
+        if "stretch" in results[key]:
+            stretches.append(results[key]["stretch"])
+        if "smear" in results[key]:
+            smears.append(results[key]["smear"])
+
+    # Input parameters used for every morph
+    inputs = [None, None]
+    inputs[0] = f"# Morphed file: {morph_file.name}"
+    inputs[1] = "\n# Input morphing parameters:"
+    inputs[1] += f"\n# scale = {opts.scale}"
+    inputs[1] += f"\n# stretch = {opts.stretch}"
+    inputs[1] += f"\n# smear = {opts.smear}"
+    input_header = f"{inputs[0]}{inputs[1]}\n"
+
+    # Verbose to get output for every morph
+    verbose_header = ""
+    if opts.verbose:
+        # Output for every morph (information repeated in a succint table below)
+        for key in results.keys():
+            outputs = f"\n# Target: {key}\n"
+            outputs += "# Optimized morphing parameters:\n"
+            outputs += "\n".join(f"# {param} = {results[key][param]:.6f}" for param in results[key].keys())
+            verbose_header += f"{outputs}\n"
+
+    # Table labels
+    labels = "\n# Labels: [Target]"
+    if field is not None:
+        labels += f" [{field}]"
+    for param in [["Scale", scales], ["Stretch", stretches], ["Smear", smears]]:
+        if len(param[1]) > 0:
+            labels += f" [{param[0]}]"
+    labels += " [Pearson] [Rw]\n"
+
+    # Corresponding table
+    table = ""
+    for idx in range(results_length):
+        row = f"{file_names[idx]}"
+        if field is not None:
+            row += f" {field_list[idx]}"
+        for param in [scales, stretches, smears]:
+            if len(param) > idx:
+                row += f" {param[idx]:0.6f}"
+        row += f" {pearsons[idx]:0.6f} {rws[idx]:0.6f}"
+        table += f"{row}\n"
```

Comment:
Terminal output for multiple morphs is a table summarizing details of the morph

![image](https://github.com/diffpy/diffpy.pdfmorph/assets/59151395/ee3ca9af-4f24-4976-a473-a8566b4fc580)

Reply:
this looks really great!